facturacion_mexico/facturacion_fiscal/tests/test_generador_templates_fiscal.py

```
@@ -0,0 +1,428 @@
+
+
+class TestITTDeduplicacion(FrappeTestCase):
+	"""
+	Verifica que _crear_o_actualizar_itt deduplica por account_head.
+
+	Un ITT no puede tener la misma cuenta dos veces (ERPNext bloquea con
+	"entered twice in Item Tax"). Cuando dos roles apuntan a la misma cuenta
+	debe generarse una sola fila — la primera en orden.
+	"""
+
+	def _run_itt(self, taxes_config, mapeo_cuentas):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_run_itt`

(ANN202)